### PR TITLE
Fix an wrong positioning of the rect of Overview

### PR DESCRIPTION
**Pull request by Vane11ope**
when either of the scrollbar in Graph is not visible, the range of the rectangle in Overview is wrong. And that is gonna be fixed by this.

### CODE REVIEW

File: src/widgets/OverviewView.h [hunk 1]
```diff
@@ -26,6 +26,13 @@ class OverviewView : public GraphView
      */
     QRectF rangeRect;
 
+    /**
+     * @brief offset for the rect which is put when either of the scrollbar of
```
Comment (ITAYC0HEN) [inline src/widgets/OverviewView.h]: Use /*!
And \brief

**Comment by ITAYC0HEN** [comment]
Looks good to me. Will fetch locally and test soon

File: src/widgets/OverviewView.h (hunk 1 above)
Comment (Vane11ope) [reply src/widgets/OverviewView.h]: oh that's simply a mistake and that's basically everywhere. let me fix so all over quickly :)

**Comment by Vane11ope** [comment]
I think this should be good enough for now to be merged.

**Pull request merged**